server/routes/componentRequests.ts

```diff
@@ -29,9 +152,19 @@ export default function routes({ componentNameService, serviceCatalogueService,
     res.send(componentRequests)
   })
 
-  router.get('/:repo_name', async (req, res) => {
+  router.get('/:repo_name/:request_type', async (req, res) => {
```

Comment:
`requestType`

Reply:
Actually `repoName` also for consistency as they're actually param names and not in the URL name itself.